packages/maps/src/lib/themes/bounds.js

```diff
@@ -1,16 +1,53 @@
-export const GREATER_LONDON_ZOOM = 11;
-export const GREATER_LONDON_CENTER = [0.05, 51.5151];
+export const centerOfBounds = (bounds) => {
+	const midpoint = (min, max) => max - (max - min) / 2;
+	return [midpoint(bounds[0][0], bounds[1][0]), midpoint(bounds[0][1], bounds[1][1])];
+};
+
+// scaleBounds returns a new bounds that scales the input bounds up by the
+// specified amount { x, y } with the center as the origin.
+//
+// Given the amounts:
+// - { x: 0.1, y: 0.1 } then both width and height will be 10% bigger;
+// - { x: 0.5, y: 1 } then width will be 50% bigger and height 100% bigger;
+// - { x: 2, y: 0 } then width will be 200% bigger and height left unchanged;
+// - etc.
+export const scaleBounds = (bounds, amount = { x: 0, y: 0 }) => {
+	const width = bounds[0][0] - bounds[1][0];
+	const height = bounds[0][1] - bounds[1][1];
+
+	const widthMul = (amount.x || 0) / 2;
+	const heightMul = (amount.y || 0) / 2;
 
+	return [
+		[
+			// Bottom left point
+			bounds[0][0] - Math.abs(width * widthMul),
+			bounds[0][1] - Math.abs(height * heightMul)
+		],
+		[
+			// Top right point
+			bounds[1][0] + Math.abs(width * widthMul),
+			bounds[1][1] + Math.abs(height * heightMul)
+		]
+	];
+};
+
+export const GREATER_LONDON_ZOOM = 11;
 export const GREATER_LONDON_BOUNDS = [
-	[-0.489, 51.28],
-	[0.236, 51.686]
+	[-0.505, 51.285],
+	[0.33, 51.693]
 ];
-
-// These coordinates are in the form needed by the option to MapboxGeocoder
-export const GREATER_LONDON_BOUNDS_FLAT = [-0.489, 51.28, 0.236, 51.686];
```

Comment:
Why is this deleted?

Reply:
You can use `GREATER_LONDON_BOUNDS.flat()` (like the default options in the `<Map>` component).

Reply:
Correction, the `<Map>` component isn't using flat bounds anymore.